README.md

```diff
@@ -172,6 +172,17 @@ clang-format can be installed by command:
 - macOS: `$ brew install clang-format`
 
 
+## Usage
+`Tangle-accelerator` currently supports two kinds of APIs
+* TA originated API
+* IRI Proxy API
+
+### TA originated API
+This is the APIs originally implemented in `tangle-accelerator`. For more information, users could visit the wiki page of this repo.
+
+### IRI Porxy API
+`tangle-accelerator` supports user sends IRI Core API. User doesn't need to know whether there target machine is running a IRI or not. With the exactly same format of IRI API, `tangle-accelerator` would help user pass the request to IRI and pass the the response back to user.
```

Comment:
Fix the grammar. Should be "Tangle-Accelerator allows the use of IRI core APIs".

Reply:
Also, make a comprehensive list of supported proxy APIs.